src/app/Mood.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package app;
+
+import javafx.scene.control.DatePicker;
+import javafx.scene.control.Slider;
+
+/**
+ *
```

Comment:
Please write a description of the class and anything special needed to use it. That's one of the key things the professor will be looking for.